packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VolumetricViewer/VolumetricViewerWrapper.js

```diff
@@ -4,7 +4,9 @@ import { MobXProviderContext } from 'mobx-react'
 import { useContext } from 'react'
 
 const DEFAULT_HANDLER = () => {}
-const VolumetricViewer = lazy(() => import('@zooniverse/subject-viewers/VolumetricViewer'))
+const VolumetricViewer = lazy(() => import('@zooniverse/subject-viewers/VolumetricViewer').then(module => {
+  return { default: module.default.Full }
+}))
```

Comment:
While this lazy loading works, I want to ask about the syntax. In all other FEM packages, components like Full are surfaced as available default exports instead of chaining `.then()`. For example, from lib-subject-viewers you can still surface it like this in index.js:
`export { default as VolumetricViewerFull } from './VolumetricViewer/VolumetricViewerFull.js`. It removes the need for a barrel file like ` export default {Full, Preview, View}`.

Then, here in lib-classifier the import statement would be:
`const VolumetricViewerFull = lazy(() => import('@zooniverse/subject-viewers/VolumetricViewerFull')`

I don't think chaining `.then()` has an advantage here, and same thoughts about using it to get "Preview" in Media.